docs/sources/integrations/jira/index.md

```diff
@@ -175,4 +175,7 @@ For more information on Jira REST API, refer to [Jira REST API documentation](ht
 {{% docs/reference %}}
 [user-and-team-management]: "/docs/oncall/ -> /docs/oncall/<ONCALL VERSION>/user-and-team-management"
 [user-and-team-management]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/oncall/user-and-team-management"
+
+[outgoing-webhooks]: "/docs/oncall/ -> /docs/oncall/<ONCALL VERSION>/outgoing-webhooks"
```

Comment:
without these defenitions links lead to https://grafana.com/docs/oncall/latest/integrations/